packages/learn-card-core/src/wallet/plugins/ceramic/ceramic.ts

```diff
@@ -46,7 +49,8 @@ export const getCeramicPlugin = async <URI extends string = ''>(
     const publishContentToCeramic = async (
         content: any,
         metadata: TileMetadataArgs = {},
-        options: CreateOpts = {}
+        options: CreateOpts = {},
+        encryption: CeramicEncryptionParams | undefined
```

Comment:
```suggestion
        encryption?: CeramicEncryptionParams
```

nit: Clears up that ESLint error